openpype/hosts/houdini/plugins/create/create_arnold_rop.py

```diff
@@ -0,0 +1,76 @@
+from openpype.hosts.houdini.api import plugin
+from openpype.lib import EnumDef
+
+
+class CreateArnoldRop(plugin.HoudiniCreator):
+    """Arnold ROP"""
+
+    identifier = "io.openpype.creators.houdini.arnold_rop"
+    label = "Arnold ROP"
+    family = "arnold_rop"
+    icon = "magic"
+    defaults = ["master"]
+
+    # Default extension
+    ext = "exr"
+
+    def create(self, subset_name, instance_data, pre_create_data):
+        import hou
+
+        # Remove the active, we are checking the bypass flag of the nodes
+        instance_data.pop("active", None)
+        instance_data.update({"node_type": "arnold"})
+
+        # Add chunk size attribute
+        instance_data["chunkSize"] = 1
+        # Submit for job publishing
+        instance_data["farm"] = True
+
+        instance = super(CreateArnoldRop, self).create(
+            subset_name,
+            instance_data,
+            pre_create_data)  # type: plugin.CreatedInstance
+
+        instance_node = hou.node(instance.get("instance_node"))
+
+        # Hide Properties Tab on Arnold ROP since that's used
+        # for rendering instead of .ass Archive Export
+        parm_template_group = instance_node.parmTemplateGroup()
+        parm_template_group.hideFolder("Properties", True)
+        instance_node.setParmTemplateGroup(parm_template_group)
+
+        ext = pre_create_data.get("image_format")
+
+        filepath = "{}{}".format(
```

Comment:
Maybe this could be done in one format only?
```py
        filepath = "{renders_dir}{subset_name}/{subset_name}.$F4.{ext}".format(
            renders_dir=hou.text.expandString("$HIP/pyblish/renders/"),
            subset_name=subset_name,
            ext=ext,
        )
  ```